docs/diagrams/interfaces/Organization/Organization.pu

```diff
@@ -0,0 +1,18 @@
+@startuml Organization Diagram
```

Comment:
manca una sua implementazione

Reply:
Le interfacce in typescript sono un pò diverse da Java, possono essere istanziate anche da sole